appveyor.yml

```diff
@@ -21,7 +21,7 @@ environment:
   matrix:
     - CARGO_HOME: "c:\\cargo"
       hab_build_action: "test;build"
-      hab_components: "core;http-client;builder-protocol;builder-depot-client;common;win-users"
+      hab_components: "core;http-client;builder-protocol;builder-depot-client;common;win-users;sup"
```

Comment:
ideally yes but it was intermittently failing 1 `cargo test` and should address that separately.